astropy/visualization/wcsaxes/coordinate_helpers.py

```diff
@@ -109,6 +109,7 @@ def __init__(self, parent_axes=None, parent_map=None, transform=None,
         self.ticklabels = TickLabels(self.frame,
                                      transform=None,  # display coordinates
                                      figure=parent_axes.get_figure())
+        self.ticklabels.parent_coords = self
```

Comment:
Ideally one would hand this as an argument to `TickLabels`, but I can't add a parameter to `TickLabels.__init__` whilst maintaining backwards compatibility.